docs/docs/integrations/platforms/google.mdx

```diff
@@ -170,33 +248,91 @@ documents = docai_wh_retriever.get_relevant_documents(
 ```
 
 ## Tools
+
+### Google Drive
+
+We need to install several python packages.
+
+```bash
+pip install google-api-python-client google-auth-httplib2 google-auth-oauthlib
+```
+
+See a [usage example and authorizing instructions](/docs/integrations/tools/google_drive).
+
+```python
+from langchain.utilities.google_drive import GoogleDriveAPIWrapper
+from langchain.tools.google_drive.tool import GoogleDriveSearchTool
+```
+
+### Google Places
+
+We need to install a python package.
+
+```bash
+pip install googlemaps
+```
+
+See a [usage example and authorizing instructions](/docs/integrations/tools/google_places).
+
+```python
+from langchain.tools import GooglePlacesTool
+```
+
 ### Google Search
 
-- Install requirements with `pip install google-api-python-client`
+We need to install a python package.
+
+```bash
+pip install google-api-python-clien
+```
+
 - Set up a Custom Search Engine, following [these instructions](https://stackoverflow.com/questions/37083058/programmatically-searching-google-in-python-using-custom-search)
 - Get an API Key and Custom Search Engine ID from the previous step, and set them as environment variables `GOOGLE_API_KEY` and `GOOGLE_CSE_ID` respectively
 
-There exists a `GoogleSearchAPIWrapper` utility which wraps this API. To import this utility:
-
 ```python
 from langchain.utilities import GoogleSearchAPIWrapper
 ```
 For a more detailed walkthrough of this wrapper, see [this notebook](/docs/integrations/tools/google_search.html).
 
 We can easily load this wrapper as a Tool (to use with an Agent). We can do this with:
+
 ```python
 from langchain.agents import load_tools
 tools = load_tools(["google-search"])
 ```
 
-## Document Transformer
-### Google Document AI
+### Google Serper
```

Comment:
think this is https://serper.dev (uses google under the hood but not developed by google)

Reply:
Correct - This shouldn't be included

Reply:
"All functionality related to Google Cloud Platform and other `Google` products."

Users don't care if the access is implemented by the Google library or by a third-party library; it is an "internal" detail. They just want to access Google Search. 
For example, the same as with `UnstructuredWordDocumentLoader`, `unstructured` is not a Microsoft product but a layer between a user and Microsoft Word.

Reply:
i think in the providers/platforms section specifically folks are looking for tools created by those providers/platforms. if we do want to keep stuff like this here then i'd strongly prefer a separate `3rd Party Integrations` section at the bottom, it's important for folks to know this isn't provided by Google.